components/sensors/pressure/pressureSensor.c

```diff
@@ -110,6 +116,15 @@ COMPONENT_INIT
 {
     // Use the periodic sensor component from the Data Hub to implement the timers and the
     // interface to the Data Hub.
-    psensor_Create("pressure", DHUBIO_DATA_TYPE_NUMERIC, "kPa", SamplePressure, NULL);
-    psensor_Create("pressure/temp", DHUBIO_DATA_TYPE_NUMERIC, "degC", SampleTemperature, NULL);
+    psensor_Create(PRESSURE_PREFIX_NAME "pressure",
+                   IO_DATA_TYPE_NUMERIC,
+                   "kPa",
+                   SamplePressure,
+                   NULL);
+
+    psensor_Create(PRESSURE_PREFIX_NAME "pressure/temp",
```

Comment:
That's a weird name, isn't it? "pressure/temp". I know it isn't your name.

Reply:
Sorry, no matter how wierd it is, we can't change it.  Customers and the backend are already aware of this path

Reply:
Yes, I understand, just a passing comment 👍. 
I did want to talk to Lars whether there was any way of having pSensor automatically determine the app name prefix.

Reply:
I guess the periodic sensor module could use a trick like the one above ^^ but I don't quite see who would use it.  
When psensor is built normally, using the IO api, the app name will be prepended by the data hub.  When built for Octave, using the admin api, the paths must be forced to something other than the app name.  For example, the unified app name will be "octaved" but the location resources still need to be named "/app/location/coordinates/...".  What do you think?